site/docs/tasks/control-plane-security.md

```diff
@@ -43,9 +43,9 @@ spec:
 
 #### Adding TLS Secrets
 
-The manager/connectors certificates are kept in Kubernetes secret:
+The manager/connectors certificates are kept in Kubernetes secret of type tls which are mounted to the manager/connectors pods in fybrik deployment.
 
-For each component copy its certificate into a file names tls.crt. Copy the certificate key into a file named tls.key.
+For each component copy its certificate into a file named tls.crt. Copy the certificate key into a file named tls.key.
```

Comment:
may be to mention, that the name can be redefined

Reply:
Done.